integration-tests/src/pipeline_tests/audio_only.rs

```diff
@@ -427,3 +540,154 @@ pub fn single_input_aac_mp4() -> Result<()> {
 
     Ok(())
 }
+
+/// Single frequency input that changes after 5 seconds. Input starts streaming 2 seconds before
+/// start request with offset set to 2 seconds. This should result in 2 seconds of silence, 5
+/// seconds of lower frequency and higher frequency after that time.
+#[test]
+fn audio_offset_variable_frequency() -> Result<()> {
```

Comment:
Instead of variable frequency, I would use `two_frequencies`, `two_frequencies_input` or `frequency_change`

However, focusing on that in the name is already a mistake. You are using this specific input to test something. Test name should reflect the goal, and using this specific input does not change what you are testing, it just makes it easier to detect errors.